### PR TITLE
User can delete selected QR codes.

### PR DESCRIPTION
**Pull request by Kiptong**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,10 +23,9 @@ app.post('/generate', (req, res) => {
       .insert(qrCode)
       .into('qrcodes')
 
-    query.then(() => console.log('done!'))
+    query
+      .then(() => res.json(qrCode))
```
Comment (victormejia) [inline index.js]: you should return here in the response the object that knex gives you. To do so, chain a `.returning('*')` after the `.into()`

File: index.js [hunk 2]
```diff
@@ -57,6 +56,16 @@ app.get('/qrcards', (req, res) => {
     .catch((err) => res.status(500).json({error: 'Error in creating QR card.'}))
 })
 
+app.delete('/deleteqr/:id', (req, res) => {
+  const query = knex('qrcodes')
+    .where('id', '=', req.params.id)
+    .del()
+
+  query
+    .then((data) => res.json(data))
```
Comment (victormejia) [inline index.js]: does the query actually give you back any data?

File: index.js [hunk 3]
```diff
@@ -57,6 +56,16 @@ app.get('/qrcards', (req, res) => {
     .catch((err) => res.status(500).json({error: 'Error in creating QR card.'}))
 })
 
+app.delete('/deleteqr/:id', (req, res) => {
```
Comment (victormejia) [inline index.js]: this should be the same path `qrcodes/:id`

**Review by victormejia — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (Kiptong) [reply index.js]: Honestly I'm really not sure if that even needs to be there.

File: index.js [hunk 4]
```diff
@@ -44,7 +44,7 @@ app.put('/qrcards/:id', (req, res) => {
       .returning('*')
 
     query
-      .then((data) => res.json(data))
+      .then((data) => console.log(res.json(data)))
```
Comment (victormejia) [inline index.js]: you are not responding back here...

File: index.js (hunk 2 above)
Comment (victormejia) [reply index.js]: then you should probably just reply with a 200 status code: `res.sendStatus(200)`

**Review by victormejia — CHANGES_REQUESTED** [review]

**Review by victormejia — APPROVED** [review]

**Pull request merged**